README.md

```diff
@@ -1,28 +1,108 @@
+   - System applies behavior changes to apps with targetSdkVersion at or above the OS version introducing these changes.
+   - New apps should target the latest version; existing apps should update when feasible.
+   - Retrieve the targetSdkVersion at runtime with android_get_application_target_sdk_version() in API level 24 and later.
+   - See https://developer.android.com/ndk/guides/sdk-versions for more information.
+
+
+**versionCode**
+   - A positive integer that serves as the internal version number of the app.
+   - Used to differentiate between newer and older versions, with higher numbers indicating more recent versions.
+   - Not visible to users, as the versionName attribute is used for display.
+   - Prevents downgrading by blocking the installation of APKs with lower versionCodes than the one installed.
+   - Important to increment for each app update.
+   - Maximum allowable value on Google Play: 2100000000.
+   - Reuse of versionCode for Play Store uploads is not permitted.
+   - See https://developer.android.com/studio/publish/versioning for more information.
+
+
+**versionName**
+   - The version number shown to users, specified as a string.
+   - Flexible format, commonly used as <major>.<minor>.<point> or other version identifiers.
+   - The primary version identifier for end users.
+   - See https://developer.android.com/studio/publish/versioning for more information.
+
+
+**package**
+   - Represents the app's unique application ID, formatted as a Java package name.
+   - Can include uppercase and lowercase letters, numbers, and underscores, but must start with a letter.
+   - Modifying this value essentially creates a new application, impacting updates and data transfer.
+   - In AGP 7.3+, avoid setting this directly in the source manifest.
+   - See https://developer.android.com/guide/topics/manifest/manifest-element for more information.
+
+
+## Coming soon
+
+**compileSdkVersion**
+   - This property doesn't impact NDK builds as API availability for NDK is determined by minSdkVersion.
+   - In NDK, C++ symbols are resolved at library load time, unlike the lazy resolution in Java.
+   - Utilizing symbols not present in the minSdkVersion can lead to library loading failures on OS versions lacking the newer API.
+   - Recommended approach for new apps is to use the latest available version. For existing apps, update to the latest version as per development needs.
+   - See https://developer.android.com/ndk/guides/sdk-versions for more information.
+
+
+**compileSdkVersionCodename**
+   - Reflects the development codename of the Android framework used for compiling the app.
+   - Compile-time equivalent of Build.VERSION.CODENAME.
+   - See https://developer.android.com/reference/android/content/pm/ApplicationInfo#compileSdkVersionCodename for more information.
+
+
+**platformBuildVersionCode**
+   - Description unavailable.
+
+
+**platformBuildVersionName**
+   - Description unavailable.
+
```

Comment:
Since the "Coming soon" section promises something that I didn't even know would come in the future, maybe it's better if you just add this once the respective feature is contributed? Otherwise if you suddenly don't have time to contribute, that would leave the project with a promise that never gets fulfilled.

Reply:
Thank you for pointing this out. This hadn't occurred to me at the time but you're right. I'll remove the "Coming soon" section.